webpack/scenes/ContentViews/Details/Filters/CVRpmFilterContent.js

```diff
@@ -0,0 +1,115 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { shallowEqual, useSelector } from 'react-redux';
+import { TableVariant } from '@patternfly/react-table';
+import { Tabs, Tab, TabTitleText } from '@patternfly/react-core';
+import { STATUS } from 'foremanReact/constants';
+import { translate as __ } from 'foremanReact/common/I18n';
+
+import onSelect from '../../../../components/Table/helpers';
+import TableWrapper from '../../../../components/Table/TableWrapper';
+import {
+  selectCVFilterRules,
+  selectCVFilterRulesStatus,
+} from '../ContentViewDetailSelectors';
+import { getCVFilterRules } from '../ContentViewDetailActions';
+
+const CVRpmFilterContent = ({ filterId, inclusion }) => {
+  const response = useSelector(state => selectCVFilterRules(state, filterId), shallowEqual);
+  const status = useSelector(state => selectCVFilterRulesStatus(state, filterId), shallowEqual);
+
+  const [rows, setRows] = useState([]);
+  const [metadata, setMetadata] = useState({});
+  const [searchQuery, updateSearchQuery] = useState('');
+  const [activeTabKey, setActiveTabKey] = useState(0);
+  const loading = status === STATUS.PENDING;
+
+  const columnHeaders = [
+    __('RPM name'),
+    __('Architecture'),
+    __('Versions'),
+  ];
+
+  const versionText = (rule) => {
+    const { version, min_version: minVersion, max_version: maxVersion } = rule;
+
+    if (rule.version) return `Version ${version}`;
+    if (rule.min_version && !rule.max_version) return `Greater than version ${minVersion}`;
+    if (!rule.min_version && rule.max_version) return `Less than version ${maxVersion}`;
+    if (rule.min_version && rule.max_version) {
+      return `Between versions ${rule.min_version} and ${rule.max_version}`;
```

Comment:
Are `min_version` and `max_version` inclusive?  The variable names `min_version` and `max_version` imply inclusive (e.g. `min_version` of 3.0 means "3.0 or newer" rather than "newer than 3.0"), but the texts imply non-inclusive ("newer than 3.0").

Also I think "newer / older" works better than "greater / less than"

Reply:
I think this whole block is adapted from the old page: 
https://github.com/Katello/katello/blob/b3ec5d421be54305bfdc618e4849d4e4ffaa689a/engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/filters/views/package-filter-details.html#L87

The way the filters are created, if only min_version is provided, it's a greater than filter, if only max_version then it's a less than filter, and for a between filter, both are set

Reply:
Somewhere in our backend code, there is a place where it actually reads `min_version` and `max_version` and uses the `>` or `>=` operator.  Let's find out which it is and make sure the language here matches that.

Reply:
Based on discussions, settling on the old wordings for versions. Also, this should become easier to follow once we the "Show matching content" button on the filters page.